lib/matplotlib/widgets.py

```diff
@@ -1332,37 +1332,47 @@ class MultiCursor(Widget):
     Provide a vertical (default) and/or horizontal line cursor shared between
     multiple axes.
 
-    For the cursor to remain responsive you must keep a reference to
-    it.
+    You can turn on/off one horizontal or one vertical in axes under the mouse
```

Comment:
Thanks for the fix on the example.  But I don't understand this new option from the docstring.   What is the use case here?   It seems that for the oneVert/Horiz you are redrawing the line each time.  That doesn't seem very efficient compared to just setting the x-data...

Reply:
You are right,i will fix it later

Reply:
I have fixed it, and add more usage example in docstring.